pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTableRestletResource.java

```diff
@@ -508,22 +516,28 @@ public String checkTableConfig(String tableConfigStr) {
           + "Validate given table config and schema. If specified schema is null, attempt to retrieve schema using the "
           + "table name. This API returns the table config that matches the one you get from 'GET /tables/{tableName}'."
           + " This allows us to validate table config before apply.")
-  public String validateTableAndSchema(TableAndSchemaConfig tableSchemaConfig) {
+  public String validateTableAndSchema(
+      TableAndSchemaConfig tableSchemaConfig,
+      @ApiParam(value = "comma separated list of validation skip") @QueryParam("typesToSkip") String typesToSkip) {
```

Comment:
whichever query param name you choose, lets keep it consistent across the 3 methods (typesToSkip here but validationTypesToSkip in other 2)

Reply:
personally I liked "skipValidations" name better. But up to you. 
The description in ApiParam is different in all 3 methods, make them the same too

Reply:
oh, good catch (thanks Intellij refactor :-P )
for the naming i think @Jackie-Jiang and I chatted about `skipValidations` seems to indicate it should be a boolean. but couldn't find a name that's short enough for the queryparam. let's leave it as `validationTypesToSkip` for now.